dts/bindings/base/pm.yaml

```diff
@@ -30,3 +30,8 @@ properties:
     description: |
       Automatically configure the device for runtime power management after the
       init function runs.
+
+  zephyr,disabling-power-states:
+    type: phandles
+    description: |
+      List of power states that will disable this device power.
```

Comment:
just out of curiosity, why did you choose to list the disabling instead of the enabling states

Reply:
Because that is the information that you need. You want to know which states cause power loss to a device so you can put a constraint in these states to forbid the SoC to use them when it is idle.